.env

```
@@ -0,0 +1,3 @@
+VITE_API_BASE_URL=https://api.proovy.ai.kr
+VITE_KAKAO_CLIENT_ID=8280c1be4f59d21d6bac83903d667f24
+VITE_KAKAO_REDIRECT_URI=http://localhost:5173/auth/kakao/callback
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)